src/screencast/screencast.ts

```diff
@@ -176,6 +182,14 @@ export class Screencast {
         return selectedOption.getAttribute('touch') === 'true' || selectedOption.getAttribute('mobile') === 'true';
     }
 
+    private deviceUserAgent() {
+        if (this.deviceSelect.value.toLowerCase() === 'desktop') {
+            return navigator.userAgent;
```

Comment:
I realize this is incorrect. We want the browser's initial userAgent not the VSCode managed webview's.

Reply:
For that, we should be able to unset the override by passing the empty string.

Reply:
Yep!